clientapi/ratelimit/rt_failed_login_test.go

```diff
@@ -0,0 +1,40 @@
+package ratelimit
+
+import (
+	"testing"
+	"time"
+
+	"github.com/matryer/is"
+)
+
+func TestRtFailedLogin(t *testing.T) {
+	is := is.New(t)
+	dut := NewRtFailedLogin(&RtFailedLoginConfig{
+		Enabled:  true,
+		Limit:    3,
+		Interval: 10 * time.Millisecond,
+	})
+	var (
+		can        bool
+		remaining  time.Duration
+		remainingB time.Duration
+	)
+	for i := 0; i < 3; i++ {
+		can, remaining = dut.CanAct("foo")
+		is.True(can)
+		is.Equal(remaining, time.Duration(0))
+		dut.Act("foo")
+	}
+	can, remaining = dut.CanAct("foo")
+	is.True(!can)
+	is.True(remaining > time.Millisecond*9)
```

Comment:
Why specifically `time.Millisecond*9`? As far as I understood this should be dependent of `r.cfg.Interval`? Maybe I didn't get something? 🤔

Reply:
We expect `remaining` to be slightly less than 10 ms, which is interval configured for tests. Hence 9 ms, but e.g. 9990 microseconds should work as well.